pkg/apis/migration/v1alpha1/migtoken_types.go

```diff
@@ -0,0 +1,145 @@
+package v1alpha1
+
+import (
+	"context"
+	"errors"
+	"k8s.io/api/authentication/v1beta1"
+
+	authapi "k8s.io/api/authorization/v1"
+	kapi "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// MigTokenSpec defines the desired state of MigToken
+type MigTokenSpec struct {
+	IdentitySecretRef *kapi.ObjectReference `json:"identitySecretRef"`
```

Comment:
The token can be an SA token so we decided to discontinue to use the term `Identity`.  Just `SecretRef` seems sufficient.